Code/client/ModCompat/BehaviorVarsMap.cpp

```diff
@@ -0,0 +1,51 @@
+#ifdef MODDED_BEHAVIOR_COMPATIBILITY
+#include "BehaviorVarsMap.h"
+
+
+uint32_t BehaviorVarsMap::find(const uint64_t acKey, const std::string acName)
+{
+    auto map = m_map.find(acKey);
```

Comment:
a better variable name would be nice + spacing everywhere is off (empty lines)

Reply:
We have a clang format you can execute over the codebase

Reply:
Commit 2d989d98047dfa179045b7ba45333d5f17575a1b